code/modules/roguetown/roguecrafting/alchemy/reagents.dm

```diff
@@ -94,9 +90,9 @@
 	color = "#13df00"
 	metabolization_rate = REAGENTS_METABOLISM * 3
 
-/datum/reagent/medicine/strongmana/on_mob_life(mob/living/carbon/M)
+/datum/reagent/medicine/strongstam/on_mob_life(mob/living/carbon/M)
 	if(!HAS_TRAIT(M,TRAIT_NOSTAMINA))
-		M.adjust_stamina(200)
+		M.adjust_stamina(-6)
```

Comment:
this now damages stamina

Reply:
Yes, that is the intent, as stamina gets LOWER, the user can perform MORE ACTIONS, so the stam pot removing it is intentional